### ISSUE TITLE
'ColorLookupTable' object has no attribute 'lookup_color'

### ISSUE THREAD

**Issue by sopoforic** [xonsh/xonsh#1537]
Since [this prompt-toolkit commit](https://github.com/jonathanslenders/python-prompt-toolkit/commit/a9df2a24ed54eae356e622cd4cc0ddf3c4a8c008#diff-cfa50fca8c30a7fc5143ded8cec3baf1) on July 27, `lookup_color` is replaced by `lookup_fg_color` and `lookup_bg_color`, so xonsh fails to start:

```
Traceback (most recent call last):
  File "C:\Python35-32\lib\runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Python35-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python35-32\lib\site-packages\xonsh\main.py", line 265, in <module>
    main()
  File "C:\Python35-32\lib\site-packages\xonsh\main.py", line 222, in main
    shell.shell.cmdloop()
  File "C:\Python35-32\lib\site-packages\xonsh\ptk\shell.py", line 115, in cmdloop
    line = self.singleline(auto_suggest=auto_suggest)
  File "C:\Python35-32\lib\site-packages\xonsh\ptk\shell.py", line 60, in singleline
    self.styler.style_name = env.get('XONSH_COLOR_STYLE')
  File "C:\Python35-32\lib\site-packages\xonsh\base_shell.py", line 126, in styler
    self._styler = XonshStyle(env.get('XONSH_COLOR_STYLE'))
  File "C:\Python35-32\lib\site-packages\xonsh\pyghooks.py", line 309, in __init__
    self.style_name = style_name
  File "C:\Python35-32\lib\site-packages\xonsh\pyghooks.py", line 334, in style_name
    self.enhance_colors_for_cmd_exe()
  File "C:\Python35-32\lib\site-packages\xonsh\pyghooks.py", line 351, in enhance_colors_for_cmd_exe
    self._smap.update(expand_gray_colors_for_cmd_exe(self._smap))
  File "C:\Python35-32\lib\site-packages\xonsh\tools.py", line 1319, in expand_gray_colors_for_cmd_exe
    for token, idx, rgb in _get_color_indexes(style_map):
  File "C:\Python35-32\lib\site-packages\xonsh\tools.py", line 1272, in _get_color_indexes
    index = table.lookup_color(attr.color, attr.bgcolor)
AttributeError: 'ColorLookupTable' object has no attribute 'lookup_color'
```

For anyone experiencing this, installing an older version of prompt-toolkit will work: `pip install prompt_toolkit==1.0.3`.

**Comment by melund** [comment]
Thanks for reporting @sopoforic. We really ought not to use the internals of prompt toolkit directly from xonsh. These kinds of error are bound to happen. But this part of the code is a hack to get readable colors on windows. 

Anyway, can you try #1538 and see if it fixes your problems.

**Comment by sopoforic** [comment]
I can't test it right now because #1295 is happening to me since I upgraded to python 3.5.2, but the code looks good. I'll be happy to test the patch once #1295 is resolved.

**Comment by melund** [comment]
Ahh. That is annoying. Try using a conda based python installation (miniconda, or anaconda). The conda packages doesn't seem to have the same problem. Alterntively, try to set run  `set XONSH_DEBUG=0` before starting xonsh.

**Comment by astronouth7303** [comment]
I also got this problem, even with conda installs.

**Comment by gforsyth** [comment]
@astronouth7303 -- do you mean the issue in #1295 or this one?  Does @melund's fix in #1538 work for you?

**Comment by astronouth7303** [comment]
This one. I'll have to look later; it's on my work computer.
